src/main/scala/org/allenai/plugins/ScaladocGenPlugin.scala

```diff
@@ -216,7 +216,10 @@ object ScaladocGenPlugin extends AutoPlugin {
         scalaApiMappingsSetting,
         // This will pull in links from pom.xml files that sbt has generated, when the project sets
         // the apiURL key.
-        autoAPIMappings := true
+        autoAPIMappings := true,
+        // This adds the output of Unidoc to the SbtSite plugin's site directory (what gets synced
+        // to github pages).
```

Comment:
Is this producing a site index or something? I don't know these terms so can't grok the comment here.